forms/static/forms/admin/dependency_fields.js

```diff
@@ -9,19 +23,68 @@
             $('.field-victim_of').hide();
             $('.field-survivor_of').hide();
         }
+        if(short_monitor_mode > 0){
+            // Show the short monitor mode form.
+            $('.field-page_number').hide();
+            $('.field-space').hide();
+            $('.field-victim_or_survivor').hide();
+            $('.field-victim_of').hide();
+            $('.field-survivor_of').hide();
+            $('.field-is_photograph').hide();
+            $('.field-special_qn_1').hide();
+            $('.field-special_qn_2').hide();
+            $('.field-special_qn_3').hide();
+            $('.field-item_number').hide();
+            $('.field-webpage_layer_no').hide();
+            tv_check();
+            internent_check();
+        }
     });
     $(document).change(function() {
-        let newspaper_person = $('#id_newspaperperson_set-0-victim_or_survivor_1:checked').length;
-        let radio_person = $('#id_radioperson_set-0-victim_or_survivor_1:checked').length;
-        let tv_person = $('#id_televisionperson_set-0-victim_or_survivor_1:checked').length;
-        let internent_person = $('#id_internetnewsperson_set-0-victim_or_survivor_1:checked').length;
-
-        if((newspaper_person > 0) || (radio_person > 0) || (tv_person > 0) || (internent_person > 0)){
+        let short_monitor_mode = $('#short_monitor_mode:checked').length;
+        if(short_monitor_mode) {
+            $('.field-page_number').hide();
+            $('.field-space').hide();
+            $('.field-age').hide();
+            $('.field-victim_or_survivor').hide();
             $('.field-victim_of').hide();
             $('.field-survivor_of').hide();
+            $('.field-is_photograph').hide();
+            $('.field-special_qn_1').hide();
+            $('.field-special_qn_2').hide();
+            $('.field-special_qn_3').hide();
+            $('.field-item_number').hide();
+            $('.field-webpage_layer_no').hide();
+            tv_check();
+            internent_check();
```

Comment:
Small optimization/readability issue but:

 1. We already have `$('.field-age')` computed above so we can store it in a var and reuse it below,
 2. Instead of `tv_check`/`internet_check`, if I get the code correct is, we want to show age if we're filling a TV or Internet sheet. So we can just have one function that receives our var with `$('.field-age')` and checks if the sheet is TV or Internet and does the required.